src/scenic/__main__.py

```diff
@@ -6,7 +6,12 @@
 import time
 import argparse
 import random
-from importlib import metadata
+import numpy
```

Comment:
Please put imports of non-stdlib packages in a separate block. The convention we use elsewhere is:
```
imports from stdlib

imports from external packages

imports of other Scenic modules using their fully-qualified names

package-relative imports of Scenic modules (rare)
```

Reply:
Fixed!